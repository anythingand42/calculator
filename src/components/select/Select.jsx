import styled from 'styled-components';

const StyledSelect = styled.select`
  border: 1px solid #ccc;
  outline: none;
  box-sizing: border-box;
  background: none;
  min-height: 28px;
`;

export default StyledSelect;
