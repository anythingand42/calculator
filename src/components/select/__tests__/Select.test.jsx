import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import Select from '../Select';

test('Select component', () => {
  const { container } = render(<Select />);
  expect(container.firstChild).toMatchSnapshot();
});
