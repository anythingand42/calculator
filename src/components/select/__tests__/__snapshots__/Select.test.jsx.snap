// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select component 1`] = `
.c0 {
  border: 1px solid #ccc;
  outline: none;
  box-sizing: border-box;
  background: none;
  min-height: 28px;
}

<select
  class="c0"
/>
`;
